### ISSUE TITLE
You can drop `output_hidden_states=True,` in `MxbaiRerankV2.forward`

### ISSUE THREAD

**Issue by sjrl** [mixedbread-ai/mxbai-rerank#6]
The code here https://github.com/mixedbread-ai/mxbai-rerank/blob/7592f2d37db7d2dcc9627ad6dd002e8f4d4cc82b/mxbai_rerank/mxbai_rerank_v2.py#L212-L218

suggests that you're using the hidden state for the final score calculation, but it looks like just the logits are being used.

You could drop the `output_hidden_states=True` as to not return the hidden states.
